sdk/armada-protocol-common/src/common/interfaces/IArmadaPoolId.ts

```diff
@@ -59,7 +59,7 @@ export function isArmadaPoolId(
   const zodReturn = ArmadaPoolIdDataSchema.safeParse(maybeArmadaPoolId)
 
   if (!zodReturn.success && returnedErrors) {
-    returnedErrors.push(...zodReturn.error.errors.map((e) => e.message))
+    returnedErrors.push(zodReturn.error.message)
```

Comment:
Why is this changed? During the last offsite we were checking together the returned error and it looked like there was an array of errors in the Zod error.

Reply:
It's more complicated, this solution is not fully working with all possible validation errors, I'll be working on it in a separate PR next after this. I'll be refactoring all those globally creating a zod validation helpers to clean up repeated code